src/db/http/connection.js

```diff
@@ -17,7 +30,7 @@ const httpOptions = {
  */
 const validateHttp = async () => {
   try {
-    await rp.head({ ...{ uri: baseUri }, ...httpOptions });
+    await rp.get({ ...{ uri: baseUri }, ...httpOptions });
```

Comment:
Ideally, we should use HEAD request for heath check but Docusign API doesn't support this method.